src/apprt/gtk/notebook_adw.zig

```diff
@@ -23,6 +23,7 @@ pub const NotebookAdw = struct {
         assert(adwaita.enabled(&app.config));
 
         const tab_view: *c.AdwTabView = c.adw_tab_view_new().?;
+        c.gtk_widget_add_css_class(@ptrCast(@alignCast(tab_view)), "notebook");
```

Comment:
I'm shocked this is needed, but I'm guessing it's to unify between GTK/Adw? Btw I'm fine with generic "notebook" here because I think the specificity will come from the parent containing the notebook.

Reply:
It's so that I can distinguish separators inside the notebook (i.e. between splits) and separators elsewhere in the UI (i.e. in menus).

Reply:
No I mean I'm more shocked that you can't select on element type without a class, i.e.:

```
.ghostty Notebook {}
```

But I guess GTK doesn't expose them like this?

Reply:
Oh, yeah you can select based on element type. Maybe there's a better way to do it but I'm not a CSS expert by any means.

Reply:
Ah, I remember, this _is_ because of the gtk/adw split - we use different elements depending on the flavor so we need a class.

Reply:
Yep that's what I figured, makes sense!